aperturedb/Utils.py

```diff
@@ -473,7 +473,8 @@ def count_connections(self, connections_class, constraints=None) -> int:
 
         return total_connections
 
-    def add_descriptorset(self, name: str, dim: int, metric="L2", engine="FaissFlat",
+    def add_descriptorset(self, name: str, dim: int, metric=["L2", "CS", "IP"],
```

Comment:
I don't think we should do this by default. Usually, in real use cases, you use a single metric. The embeddings are generated in such a way that a specific metric will work best. 

This would, by default, replicate the data in 6 different faiss indexes.

Reply:
fair enough. I will set defaults as : "HNSW", "CS"